RegionXComponents/src/styles/_variables.scss

```diff
@@ -0,0 +1,24 @@
+$yellowDark: #F59E0B;
+$yellowLight3: #FEF3C7;
+$greenDark: #1A8245;
+$greenLight6: #DAF8E6;
+$orangeDark: #E1580E;
```

Comment:
Let’s use the same colors as specified in the design. You should be able to get the hex codes easily from the Figma file.

Reply:
@Szegoo i imported colors from figma, but some colors used in the design-system, don't match any colors from COLOR section, would you like me to stick to the design of components, or use most similar color from the COLOR section?

Reply:
We should use the colors from the design of the components.